project1/templates/login.html

```diff
@@ -49,6 +51,56 @@
         }
 </style>
 {% endblock %}   
+{% block scripts%}
+<script type = "text/javascript"> 
+// Book Form
+function apiBook(e) {
+var isbn = e.id;
+  var url = "/api/book?isbn="+isbn ;
+  var xhr = new XMLHttpRequest();
+  
+  console.log(isbn);
+  xhr.open("GET", url, true);
+  xhr.setRequestHeader("Content-type", "application/json");
+  let json = JSON.stringify ({
+    "isbn" : isbn
+  });
+  xhr.send();
+  xhr.onload = () => {
+    if (xhr.status == 200) {
+      var data = JSON.parse(xhr.responseText);
+      console.log(data);
+      var htmltext = "<div class='card' style = 'width:50rem;'><ul>";
+      htmltext= htmltext + "<img src = 'http://covers.openlibrary.org/b/isbn/ "+ data.isbn + ".jpg ' width = '250' height = '250' alt = "+ data.title +"/> <li class ='list-group-item'><b>Isbn:</b>" + data.isbn+ "</li>" + "<li class = 'list-group-item'><b>Title:</b>"+ data.title+"</li>"+        
+        "<li class = 'list-group-item'><b>Author:</b>"+data.author+"</li>"+
+        "<li class = 'list-group-item'><b>Year:</b>"+ data.year+"</li>"
+    
+      htmltext=htmltext + "</ul></div>"
+      console.log(data.reviews);
+      // htmltext = htmltext + "<p id = '1416949658'>"+data.isbn +"</p><ul>";
+      htmltext=htmltext+"<div><h1> Reviews</h1>"+"<tr>"
+      for(var i = 0; i < data.reviews.length; i++) {
+      htmltext=htmltext+"<h1 style = 'text-align:center'></h1>"+"<div class='card' style='width: 50rem; align:center'>"+
+      "<ul class='list-group list-group-flush'>"+"<li class='list-group-item'><b text-primary>&#x1F464: </b>"+data.reviews[i]["email"]+"</li>"+
+      "<li class='list-group-item'><b text-primary>&#11088: </b>"+data.reviews[i]["rate"]+"</li>" +"<li class='list-group-item'><b text-primary>&#x1F4AC: </b>"+data.reviews[i]["review"]+"</li>"
+      }
+      htmltext = htmltext + " </ul></div></tr></div>"
+      
+           
+      document.getElementById("htmltext").innerHTML = htmltext;
+    }
+    else if(xhr.status == 404) {
+      console.log("Error:  No Book Found");
```

Comment:
Please display the error message to the user as well. Don't just print in the console